src/applications/disability-benefits/all-claims/tests/components/ConfirmationPoll.unit.spec.jsx

```diff
@@ -267,4 +275,34 @@ describe('ConfirmationPoll', () => {
       ]);
     });
   });
+
+  describe('ConnectedConfirmationPoll', () => {
+    it('should return areConfirmationEmailTogglesOn as true when confirmationEmailFeature toggles on', () => {
+      mockApiRequest(successResponse.response);
+      const togglesOnState = {
+        featureToggles: {
+          /* eslint-disable camelcase */
```

Comment:
Disabled camelcase linting rule because the feature toggle names are snackcase, and we will be removing these feature toggles in the very near future.